pkg/ocm/helpers.go

```diff
@@ -210,6 +212,28 @@ func (c *Client) GetCurrentAccount() (*amsv1.Account, error) {
 	return response.Body(), nil
 }
 
+// MarkLinkedRoles gets a slice of roles and populates the role's 'Linked' attribute with 'Yes'/'No'
+func (c *Client) MarkLinkedRoles(roles []aws.Role) error {
+	orgID, _, err := c.GetCurrentOrganization()
+	if err != nil {
+		return fmt.Errorf("failed to get organization account: %v", err)
+	}
+	linkedRoles, err := c.GetLinkedRoles(orgID)
```

Comment:
You need to pass the label as parameter to this func. 
This looks for `sts_ocm_role` label, but you need to look for `sts_user_role`